euth/ideas/templates/euth_ideas/idea_list.html

```diff
@@ -2,88 +2,90 @@
 {% load i18n rules idea_tags %}
 
 {% block phase_content %}
-<div class="idea-list">
-    {% has_perm 'euth_ideas.propose_idea' request.user view.module as add_button_enabled %}
-    {% if add_button_enabled %}
-    <a href="{% url 'idea-create' slug=view.module.slug %}" class="idea-list-button" id="idea-create">
-        <i class="fa fa-plus idea-list-button-icon" aria-hidden="true"></i>
-        {% trans 'Create Proposal' %}
-    </a>
-    {% endif %}
+<div class="container-medium">
```

Comment:
@slomo yes, this matter has been cleared. I've used the different classes fully intentional.